data/mods/Aftershock/items/comestibles/alienfood.json

```diff
@@ -132,5 +173,406 @@
+    "description": "A brain butchered from an alien organism.  It looks somewhat similar to its earthly analogues, barring its light-blue coloration and gland riddled frontal lobe.",
+    "color": "light_gray",
+    "healthy": -15,
+    "price_postapoc": 10,
+    "quench": 10,
+    "fun": -20,
+    "parasites": 15,
+    "calories": 78,
+    "vitamins": [ [ "vitA", 0 ], [ "vitC", 2 ], [ "calcium", 1 ], [ "iron", 8 ], [ "vitB", 54 ], [ "mutant_toxin", 25 ] ]
+  },
+  {
+    "id": "alien_brain_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_brain",
+    "looks_like": "offal_cooked",
+    "name": { "str_sp": "cooked alien brains" },
+    "description": "Fried and scrambled, this doesn't look the most appetizing, but it's edible (you hope).",
+    "price_postapoc": 25,
+    "parasites": 0,
+    "healthy": -2,
+    "fun": -8,
+    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_kidney",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal",
+    "name": "raw alien kidney",
+    "description": "You are guessing here, but this is likely the kidney of the animal.  The ammonia smell was the biggest hint.",
+    "color": "brown",
+    "healthy": -8,
+    "price_postapoc": 25,
+    "quench": -5,
+    "calories": 54,
+    "vitamins": [ [ "vitA", 6 ], [ "vitC", 2 ], [ "calcium", 0 ], [ "iron", 8 ], [ "vitB", 155 ], [ "mutant_toxin", 25 ] ]
+  },
+  {
+    "id": "alien_kidney_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_kidney",
+    "looks_like": "offal_cooked",
+    "name": "cooked alien kidney",
+    "description": "Thoroughly cooked and prepared, this alien organ has acquired the filthy ammonia smell of a poorly mopped spaceship deck.  Even if it were safe to eat, you are not sure you'd want to.",
+    "healthy": -2,
+    "price_postapoc": 25,
+    "quench": -2,
+    "fun": -3,
+    "parasites": 0,
+    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_sweetbread",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal",
+    "name": "raw alien sweetbread",
+    "description": "You got this chunk of meat from the throat of the animal; it looks different?",
+    "color": "brown",
+    "healthy": 0,
+    "price_postapoc": 25,
+    "quench": 0,
+    "calories": 133,
+    "vitamins": [ [ "vitA", 0 ], [ "vitC", 32 ], [ "calcium", 0 ], [ "iron", 6 ], [ "vitB", 20 ], [ "mutant_toxin", 25 ] ]
+  },
+  {
+    "id": "alien_sweetbread_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_sweetbread",
+    "looks_like": "meat_cooked",
+    "name": "cooked alien sweetbread",
+    "price_postapoc": 50,
+    "fun": -1,
+    "description": "Well, it's definitely not muscle, you can tell now.  It oozes a purple liquid out onto your hand as you hold it.",
+    "parasites": 0,
+    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "alien_bone",
```

Comment:
Did these get added to whatever necessary places such as "bone" LIST if it exists?  Can they be used to create charcoal in a kiln?

Reply:
I did not, I can try adding it later possibly? How would I go about adding it to the bone list?

Reply:
It's based on material, it works.
![image](https://user-images.githubusercontent.com/51099123/102682489-eacb6f00-4197-11eb-9e19-574f335bb660.png)